pkg/aws/policies.go

```diff
@@ -482,6 +482,12 @@ func (c *awsClient) FindRoleARNs(roleType string, version string) ([]string, err
 				}
 			case tags.OpenShiftVersion:
 				isTagged = true
+
+				if roleHasTag(listRoleTagsOutput.Tags, tags.ManagedPolicies, tags.True) {
```

Comment:
We can take that approach and it will simplify the code, but it won't protect developers in the future.
If someone will add a new case, and won't take into consideration managed policies, we can end up not skipping some roles.

Adding the check inside `case tags.OpenShiftVersion` will skip only version validation, but not other validations.
WDYT?

Reply:
It does make sense, I think it is fine